python/lsst/ts/scheduler/__init__.py

```diff
@@ -24,6 +24,19 @@
 except ModuleNotFoundError:
     __version__ = "?"
 
+import logging
```

Comment:
Line 23 has to be `from .version import __version__` otherwise the import will fail. We have seen that happen for other CSCs.